addons/account/models/account_bank_statement_line.py

```diff
@@ -677,6 +677,20 @@ def _seek_for_lines(self):
                 suspense_lines += line
             else:
                 other_lines += line
+
+        # A statement line should have liquidity and suspense lines - lines might not conform if the journal default
+        # accounts were changed. In this case, try to identify them using the account type and codes, raise an error?
+        if not liquidity_lines and other_lines:
+            liquidity_lines = other_lines.filtered(lambda l: l.account_id.account_type == 'asset_cash')
+            other_lines = other_lines - liquidity_lines
+
+        if not suspense_lines and len(other_lines) == 1:
+            suspense_lines = other_lines.filtered(lambda l:
+                                l.account_id.code.startswith(self.move_id.company_id.bank_account_code_prefix)
+                                and l.account_id.account_type == 'asset_current'
+                                and l.name == liquidity_lines.name)
+            other_lines = other_lines - suspense_lines
```

Comment:
This part is really dangerous because having or not a suspense line is the way to distinguish if the bank transaction needs to be reconciled or not.

Reply:
Yes indeed, but we have a similar issue if the journal's suspense account is changed - we still cannot compute that it needs to be reconciled or not